README.md

```diff
@@ -10,8 +9,9 @@
 
 **Warning: This project is in beta state. There might (and most probably will) be changes in the future to its API and working. Also, no guarantees can be made about its stability, efficiency, and security at this stage.**
 
-## Table of Contents
+## Quick link for [end-to-end tutorial](https://www.youtube.com/watch?v=Mt5468WzWaA) ▶️ ⏯️
```

Comment:
what does the end-to-end mean at the tutorial? and why is the link quick? : D also the reference is not on the link.
IMO it would be better simply saying
```suggestion
## [Tutorial video](https://www.youtube.com/watch?v=Mt5468WzWaA) ▶️ ⏯️
```
Or something similar

Reply:
yeah I don't know why I complicated it, thanks for feedback

At least I use the commit conventions 🥇

Reply:
yep, but it is related to the `docs` not the `style` though